week1/task5/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿int[] input = new[] { 1, 2, 5, 7, 2, 3, 5, 7 };
+if(input.Length % 2 != 0)
+{
+    Console.WriteLine("WARNING: the array is of odd length");
+}
+else
+{
+    int[][] splitArray = Split(input);
+    int[] result = AddArray(splitArray);
+    WriteResult(result);
+}
+
+int[][] Split(int[] input)
+{
+    int[] one = new int[input.Length / 2];
```

Comment:
instead of calling `input.Length / 2` in multiple places, you can store it in a variable and use it where needed